cmd/caa-log-checker/main.go

```diff
@@ -257,5 +257,5 @@ func main() {
 }
 
 func init() {
-	cmd.RegisterCommand("caa-log-checker", main)
+	cmd.RegisterCommand("caa-log-checker", main, nil)
```

Comment:
```suggestion
	cmd.RegisterCommand("caa-log-checker", main, &cmd.ConfigValidator{Config: &Config{}})
```

Reply:
I don't believe this takes in a config file.